src/_guides/language/language-tour_migrated.md

```diff
@@ -106,14 +109,28 @@ mind:
 
 -   Everything you can place in a variable is an *object*, and every
     object is an instance of a *class*. Even numbers, functions, and
-    `null` are objects. All objects inherit from the [Object][] class.
+    `null` are objects.
+    All objects except `null` inherit from the [Object][] class.
 
 -   Although Dart is strongly typed, type annotations are optional
     because Dart can infer types. In the code above, `number`
-    is inferred to be of type `int`. When you want to explicitly say
-    that no type is expected, use the type `Object`
-    (or `Object?` if the value can be `null`) or —
-    if you want to defer type checking until runtime —
+    is inferred to be of type `int`.
+
+-   If you enable [sound null safety][ns],
+    values can’t be null unless you say they can be.
```

Comment:
Values are what they are and `null` is just another one of them floating around in value-land. Maybe a better way to say this is, "If you enable null safety, variables can't contain null unless you say they can."

Reply:
Ha, I should fix this in /null-safety too!